tests/e2e/forms/pregnancy-visit.wdio-spec.js

```diff
@@ -0,0 +1,52 @@
+//config/default/forms/app/pregnancy_home_visit.xml
+const fs = require('fs');
+
+const utils = require('../../utils');
+const userData = require('../../page-objects/forms/data/user.po.data');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+const genericForm = require('../../page-objects/forms/generic-form.wdio.page');
+const pregnancyVisitForm = require('../../page-objects/forms/pregnancy-visit-form.wdio.page');
+const { expect } = require('chai');
+const xml = fs.readFileSync(`${__dirname}/../../../config/standard/forms/app/pregnancy_visit.xml`, 'utf8');
```

Comment:
I wonder if we should move the while xml + form document generation into the .page file.